autosklearn/smbo.py

```diff
@@ -526,6 +529,25 @@ def get_metalearning_suggestions(self):
                                   else 'dense'))
                 self.metadata_directory = metadata_directory
 
+            # If metadata directory is specified by user,
+            # then verify that it exists.
+            else:
+                if not os.path.exists(self.metadata_directory):
+                    raise ValueError('The specified metadata directory \'%s\' '
+                                     'does not exist!' % self.metadata_directory)
+                else:
+                    # There is no multilabel data in OpenML
+                    if self.task == MULTILABEL_CLASSIFICATION:
+                        meta_task = BINARY_CLASSIFICATION
+                    else:
+                        meta_task = self.task
+                        metadata_directory = os.path.join(
```

Comment:
This should not be indented this far? Right now, if the task is a multilabel classification task, the metadata_directory variable is not set.